src/etc/inc/system.inc

```diff
@@ -1923,14 +1923,15 @@ function system_reboot_sync() {
 }
 
 function system_reboot_cleanup() {
-	global $config, $cpzone;
+	global $config, $cpzone, $cpzoneid;
 
 	mwexec("/usr/local/bin/beep.sh stop");
 	require_once("captiveportal.inc");
 	if (is_array($config['captiveportal'])) {
 		foreach ($config['captiveportal'] as $cpzone=>$cp) {
-			captiveportal_radius_stop_all();
-			captiveportal_send_server_accounting(true);
+			$cpzoneid = $cp[zoneid];
+			captiveportal_disconnect_all(7); // Admin-Reboot
```

Comment:
Mostly because that's what all the other functions in captiveportal.inc do. This is the only place where it makes sense to use it but the globals aren't already set, so I wasn't sure that it was worth it to change it to use explicit parameters and then having it working in a different way.